schema/methods_directory_poc/physical/qm/pseudopotential/psp_set.json

```diff
@@ -0,0 +1,59 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "description": "Schema for a set of pseudopotential sets",
+    "properties": {
+        "type": {
+            "description": "type of pseudization",
+            "enum": [
+                "paw",
+                "us",
+                "nc"
+            ],
+            "type": "string"
+        },
+        "elements": {
+            "description": "List of elements in pseudopotential set",
+            "$ref": "../../../../core/reusable/array_of_elements.json"
+        },
+        "modelReference": {
+            "description": "Unique identifier of the model used to generate the PP set",
+            "type": "string"
+        },
+        "format": {
+            "description": "type of pseudopotential format",
+            "enum": [
+                "upf",
+                "vasp"
+            ],
+            "type": "string"
+        },
+        "name": {
+            "description": "Name for pseudopotential set",
+            "type": "string"
+        },
+        "source": {
+            "description": "explains the source of where this came from",
+            "$ref": "psp_file/origin.json#/definitions/ppSources"
+        },
+
+        "version": {
+            "description": "explains the version of where this came from",
+            "type": "string"
+        },
+        "files": {
```

Comment:
This is against the logic we used in the entity_sets to entity relationship - see https://github.com/Exabyte-io/code.js/blob/main/src/entity/set/mixins.js#L9, for instance. Currently the entities (i.e. psp_files in the current context) are pointing to the set (psp_set) by having a reference to a set `_id` in their `inSet` field.

This is a good effort, but I think we should come back to defining the way sets are handled at a later point.